scene/main/viewport.cpp

```diff
@@ -2686,6 +2682,8 @@ void Viewport::input(const Ref<InputEvent> &p_event, bool p_local_coords) {
 	}
 
 	if (is_embedding_subwindows() && _sub_windows_forward_input(p_event)) {
+		get_tree()->_call_input_pause(input_group, "_input", ev, this); //not a bug, must happen before GUI, order is _input -> gui input -> _unhandled input
+		_gui_input_event(ev);
```

Comment:
I can see the following problem with this change:
A single event is supposed to be used for a single purpose only. This change allows an event to affect something within the focused embedded window and afterwards it can affect nodes within the current viewport.